### ISSUE TITLE
Tableau web connector does not load the data and times out

### ISSUE THREAD

**Issue by xrmorgan-uh** [tableau/webdataconnector#409]
I have a custom WDC that loads a csv file from a web API. It has been working fine until yesterday the data source on Tableau server times out when refreshing. I opened the data source in Tableau desktop and it won't load the data. It runs for a few hours with no data coming back. The web API where the data source retrieves data loads fine. Nothing has changed with the custom WDC. This morning I changed the column names in the custom WDC (javascript file), the column name change gets reflected in Tableau desktop, but the data still won't load. Please help!

**Comment by xrmorgan-uh** [comment]
More information regarding the issue:

There is a total of 458325 records from the API. This does not seem to be a large data set. To rule out looping through records taking too long, I did change the loop to only loop 10 records, but the data still would not come back. It shows 'loading data...' in Tableau desktop forever without loading the data.

**Comment by theoutlander** [comment]
Can you please post some code and your logs? Has the data the API is returning changed?

**Comment by xrmorgan-uh** [comment]
Thank you for following up. It seems that the custom WDC is working now. The data source API did have a column name change. I did modify my code accordingly yesterday but the custom WDC still would not work. I decided to set up the simulator to test today and it worked in simulator. So I tried to connect in Tableau desktop and it miraculously worked. Not sure if it had anything to do with me rebooting the server that the custom WDC is hosted on. Thank you again and I think the issue is not related to the WDC library.
